assets/airports/kewr.json

```diff
@@ -0,0 +1,5706 @@
+                "COATE": ["COATE"],
+                "DIXIE": ["ELVAE", "COL","DIXIE"],
+                "ELIOT": ["ELIOT"],
+                "GAYEL": ["GAYEL"],
+                "GREKI": ["GREKI"],
+                "HAAYS": ["HAAYS"],
+                "LANNA": ["LANNA"],
+                "MERIT": ["MERIT"],
+                "NEION": ["NEION"],
+                "NEWEL": ["NEWEL"],
+                "PARKE": ["PARKE"],
+                "SHIPP": ["SHIPP"],
+                "WAVEY": ["WAVEY"],
+                "WHITE": ["ELVAE", "COL", "WHITE"],
+                "ZIMMZ": ["ZIMMZ"]
+            },
+            "draw": [
+                ["BAYYS*"],
+                ["BIGGY*"],
+                ["BREZY*"],
+                ["COATE*"],
+                ["DIXIE*"],
+                ["ELIOT*"],
+                ["GAYEL*"],
+                ["HAAYS*"],
+                ["LANNA*"],
+                ["MERIT*"],
+                ["NEION*"],
+                ["NEWEL*"],
+                ["PARKE*"],
+                ["SHIPP*"],
+                ["WAVEY*"],
+                ["WHITE*"],
+                ["ZIMMZ*"]
+            ]
+        },
+        "PORTT4": {
+            "icao": "PORTT4",
+            "name": "Port Four",
+            "rwy": {
+                "KEWR22L": [["_RWY22L01DME", "A05+"]],
+                "KEWR22R": [["_RWY22R01DME", "A05+"]]
+            },
+            "body": ["BAGGA", ["PORTT", "A25"]],
+            "exitPoints": {
+                "BAYYS": ["#261", "BAYYS"],
+                "BIGGY": ["RIETA", "SBJ", "BIGGY"],
+                "BREZY": ["#261", "BREZY"],
+                "COATE": ["#261", "COATE"],
+                "DIXIE": ["#261", "ELVAE", "COL","DIXIE"],
+                "ELIOT": ["SIRLY", "FENLY", "ELIOT","ELIOT"],
+                "GAYEL": ["#261", "GAYEL"],
+                "HAAYS": ["#261", "HAAYS"],
+                "LANNA": ["RIETA", "SBJ", "LANNA"],
+                "MERIT": ["#261", "MERIT"],
+                "NEION": ["#261", "NEION"],
+                "PARKE": ["RIETA", "SBJ", "PARKE"],
+                "WAVEY": ["#261", "WAVEY"],
+                "WHITE": ["#261", "ELVAE", "COL", "WHITE"]
+            },
+            "draw": [
+                ["_RWY22L01DME", "BAGGA"],
+                ["_RWY22R01DME", "BAGGA"],
+                ["BAGGA", "PORTT", "RIETA", "SBJ"],
+                ["BAYYS*"],
+                ["SBJ", "BIGGY*"],
+                ["BREZY*"],
+                ["COATE*"],
+                ["DIXIE*"],
+                ["PORTT", "SIRLY", "FENLY","ELIOT*"],
+                ["GAYEL*"],
+                ["HAAYS*"],
+                ["SBJ", "LANNA*"],
+                ["MERIT*"],
+                ["NEION*"],
+                ["SBJ", "PARKE*"],
+                ["WAVEY*"],
+                ["WHITE*"]
+            ]
+        },
+        "LIB5": {
+            "icao": "LIB5",
+            "name": "Liberty Five",
+            "rwy": {
+                "KEWR22L": [["_RWY22L01DME", "A05+"], ["#215", "A025"]],
```

Comment:
As with the EWR4, I'm not sure what the purpose of the altitude restriction on a heading is

Reply:
I wasn't sure how else to get the aircraft to stop their climb and wait for ATC. With the addition of the new commands feature in the spawnPatterns, this can probably go away and be handled in the spawn pattern.